senza.yaml

```diff
@@ -0,0 +1,162 @@
+    - TokeninfoUrl:
+        Description: "TokeninfoUrl"
+    - ApplicationId:
+        Description: "Application id"
+        Default: "nakadi"
+    - MintBucket:
+        Description: "Name of the mint bucket to use for this application"
+    - ScalyrAccountKey:
+        Description: "Key for Skalyr logging (optional)"
+        Default: ""
+    - InstanceType:
+        Description: "The EC2 Instance type that is used for the Nakadi instances (optional, default: t2.micro)"
+        Default: "t2.micro"
+    - InstancesMinumum:
+        Description: "The minimum number of instances of the AutoScalingGroup (optional, default: 2)"
+        Default: 2
+    - InstancesMaximum:
+        Description: "The maximum number of instances of the AutoScalingGroup (optional, default: 5)"
+        Default: 5
+    - ZookeeperKafkaNamespace:
+        Description: "Namespace in Zookeeper used by Buku (Kafka) (optional)"
+        Default: ""
+    - ExhibitorBrokers:
+        Description: "List of exhibitor brokers (comma separated)"
+    - ExhibitorPort:
+        Description: "Exhibitor port (ptional, default 8181)"
+        Default: "8181"
+    - LimitTrafficToCidr:
+        Description: "Allows request to the loadbalancer of Nakadi to only this ip address range given in cidr notation (optional, default: 0.0.0.0/0)"
+        Default: "0.0.0.0/0"
+
+SenzaComponents:
+
+  - Configuration:
+      Type: Senza::StupsAutoConfiguration # auto-detect network setup
+
+  - AppServer:
+      Type: Senza::TaupageAutoScalingGroup
+      InstanceType: "{{Arguments.InstanceType}}"
+      SecurityGroups:
+        - "Fn::GetAtt":
+          - "NakadiSecurityGroup"
+          - "GroupId"
+      IamInstanceProfile:
+        Ref: NakadiIamInstanceProfile
+      ElasticLoadBalancer: AppLoadBalancer
+      AssociatePublicIpAddress: false
+      TaupageConfig:
+        application_id: "{{Arguments.ApplicationId}}"
+        application_version: "{{SenzaInfo.StackVersion}}"
+        mint_bucket: "{{Arguments.MintBucket}}"
+        runtime: Docker
+        source: "{{Arguments.DockerImage}}"
+        health_check_path: /health
+        ports:
+          8080: 8080
+        scalyr_account_key: "{{Arguments.ScalyrAccountKey}}"
+        environment:
+          HTTP_TOKENINFO_URL: "{{Arguments.TokeninfoUrl}}"
+          OAUTH2_ACCESS_TOKEN_URL: "{{Arguments.AccessTokenUrl}}"
+          ARUHA_PUSH_OAUTH_ACTIVE: "true"
+          ARUHA_PULL_OAUTH_ACTIVE: "true"
+          ARUHA_SUBSCRIPTION_MODE: "zookeeper"
+          STREAM_HEARTBEAT_PERIOD_MS: "5000"
+          KAFKA_CLIENTS_INIT_POOL_SIZE: "4"
+          KAFKA_CLIENTS_MAX_POOL_SIZE: "16"
+          NAKADI_ZOOKEEPER_KAFKANAMESPACE:    "{{Arguments.ZookeeperKafkaNamespace}}"
+          NAKADI_ZOOKEEPER_EXHIBITOR_BROKERS: "{{Arguments.ExhibitorBrokers}}"
+          NAKADI_ZOOKEEPER_EXHIBITOR_PORT:    "{{Arguments.ExhibitorPort}}"
+      AutoScaling:
+        Minimum: "{{Arguments.InstancesMinumum}}"
```

Comment:
Senza expects to get integer here but gets string instead...
```python
Generating Cloud Formation template.. EXCEPTION OCCURRED: unorderable types: int() > str()
Traceback (most recent call last):
  File "/usr/local/bin/senza", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/senza/cli.py", line 1383, in main
    handle_exceptions(cli)()
  File "/usr/local/lib/python3.5/dist-packages/senza/cli.py", line 250, in wrapper
    func()
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/senza/cli.py", line 569, in create
    data = create_cf_template(definition, region, version, parameter, force)
  File "/usr/local/lib/python3.5/dist-packages/senza/cli.py", line 629, in create_cf_template
    data = evaluate(definition.copy(), args, account_info, force)
  File "/usr/local/lib/python3.5/dist-packages/senza/cli.py", line 236, in evaluate
    definition = componentfn(definition, configuration, args, info, force, account_info)
  File "/usr/local/lib/python3.5/dist-packages/senza/components/taupage_auto_scaling_group.py", line 40, in component_taupage_auto_scaling_group
    definition = component_auto_scaling_group(definition, configuration, args, info, force, account_info)
  File "/usr/local/lib/python3.5/dist-packages/senza/components/auto_scaling_group.py", line 150, in component_auto_scaling_group
    as_conf.get('DesiredCapacity', 1))
TypeError: unorderable types: int() > str()
```

I could envision two possible solutions:
1) Just specify 2 and 5 as Minimum and Maximum, without using Arguments
2) Try to fix senza, by explicitly casting value to integer